Documentation/Experimental/SceneUnderstanding/README_SceneUnderstanding.md

```diff
@@ -0,0 +1,99 @@
+
+![Manage MRTK Configuration](Images/ConfigureMRTKMSBuildMenu.jpg)
+
+![MSBuild enabled](Images/MRTKProjectConfiguratorMSBuildEnabled.jpg)
+
+
+## Demonstration and debugging SU
+
+### Unity demo scene
+
+Open the scene file and press play!
+
+`Assets/MixedRealityToolkit.Examples/Experimental/SpatialAwareness/DemoSpatialAwareness.unity`
+
+#### Configuring the observer service
+
+Select the 'MixedRealityTookit' game object and check the inspector.
+
+![scene understanding location in heirarchy](Images/MRTKHierarchy.png)
+![mrkt location in inspector](Images/MRTKLocation.png)
+
+These options will allow you to configure the observer on play. 
+
+If you press play at this point the example serialized scene data will be loaded and displayed in the editor (assuming the observer is configured to show 'debug' objects)
+
+These 'debug' options are meant to demonstrate some of the features of the SU DLL. It is intended that most consumers of SU will write their own scripts that subscribe to SU events, and do whatever they want with the info. In addition many observer options may be set at runtime.
+
+The provided DemoSpatialAwarenessController.cs offers examples of these things.
+
+### Example script
+
+The example script _DemoSpatialAwarenessController.cs_, demonstrates the major concepts in working with the SU service.
+
+* Subscribing to SU events
+* Handling SU events
+* Finding the nearest platform
+* Instantiating a prefab and placing it in an open spot
+
+The script uses MRKT UI to toggle service features on the controller.
+
+Turning on "Instantiate Prefabs", will demonstrate creating objects that size to fit themselves to all Scene Objects, gathered neatly under a parent object.
+
+![demo controller options](Images/Controller.png)
+
+### Built app notes
+
+You may build and deploy to HoloLens in the standard way. Once running, you'll be presented with a number of buttons to play with the features.
+
+The "Place" button will make a prefab appear on the nearest platform. (which may be behind you!) You should hear a spatial sound when this occurs to help you find it.
+
+
+Note, their are some pit falls in making queries to the observer. Misconfiguration of a fetch request result in your event payload not containing the data you expected. For example, if you don't request quads, then you won't see the occlusion mask textures. Like wise, you wont see the world mesh if you don't request meshes. The `DemoSpatialAwarenessController` script takes care of some of these dependencies, but not all.
+
+As this effort in in EXPERIMENTAL things are likely to change. Feedback requested at https://github.com/microsoft/MixedRealityToolkit-Unity/pull/7458
+
+Saved .bytes files can be accessed through the device portal at `User Folders/LocalAppData/.../LocalState/PREFIX_yyyyMMdd_hhmmss.bytes`. You can use these `bytes` for your workflow in edtor by specifying them in the observer profile.
```

Comment:
add hyperlink to [device portal](https://docs.microsoft.com/en-us/windows/mixed-reality/using-the-windows-device-portal)

"accessed through the file explorer under the [device portal] (hyperlink)"

These ".bytes" files can be utilized in editor by specifying them in the scene understanding observer profile in inspector